Tests/SwiftUICoordinatorTests/SwiftUICoordinatorTests.swift

```diff
@@ -1,11 +1,91 @@
 import XCTest
+import SwiftUI
 @testable import SwiftUICoordinator
 
 final class SwiftUICoordinatorTests: XCTestCase {
```

Comment:
`@MainActor` can be added here instead of every individual test method

Reply:
👍